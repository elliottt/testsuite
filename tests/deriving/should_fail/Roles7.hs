{-# LANGUAGE RoleAnnotations #-}

module Roles7 where

bar :: Int@P -> Int
bar = id