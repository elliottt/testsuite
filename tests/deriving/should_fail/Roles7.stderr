
Roles7.hs:5:8:
    Illegal role annotation on Int
    In the type signature for ‛bar’
